cmd/govim/testdata/scenario_default/complete_watched.txt

```diff
@@ -1,20 +1,16 @@
 # Test that ominfunc complete works where the completion is made
 # available in a file that is not loaded via the editor.
-#
-# We add in a generous sleep to ensure the watch event has been
-# handled.
 
 vim ex 'e main.go'
 cp const.go.orig const.go
-errlogmatch '&protocol\.DidOpenTextDocumentParams\{\n\S+:\s+TextDocument: protocol.TextDocumentItem{URI:"file://'$WORK/const.go
+errlogmatch '&protocol\.DidChangeWatchedFilesParams\{\n\S+:\s+Changes: \{\n\S+:\s+\{URI:"file://'$WORK/const.go'", Type:1\}'
 vim ex 'call cursor(6,16)'
 vim ex 'call feedkeys(\"i\\<C-X>\\<C-O>\\<C-N>\\<C-N>\\<ESC>\", \"x\")'
 vim ex 'w'
 cmp main.go main.go.golden
 
 # Assert that we have received no error (Type: 1) or warning (Type: 2) log messages
-# Disabled pending resolution to https://github.com/golang/go/issues/34103
-# errlogmatch -start -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:(1|2), Message:".*'
+errlogmatch -start -count=0 'LogMessage callback: &protocol\.LogMessageParams\{Type:(1|2), Message:".*'
```

Comment:
Huh, this was passing for me, and that issue was resolved, so I uncommented. Perhaps this needs to be switched to use vimexprwait.